wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -21,30 +21,41 @@ import WireCommonComponents
 import WireDataModel
 import WireSyncEngine
 
+enum Indicator {
+    case deleted
+    case edited
+    case none
+}
+
+enum TeamRoleIndicator {
+    case guest
+    case externalPartner
+    case federated
+    case service
+    case none
+}
```

Comment:
I think it's better to represent `none` with an optional value. E.g

```swift
let indicator: Indicator? = nil
```

Reply:
Done!